src/components/dragelement/index.js

```diff
@@ -141,7 +141,12 @@ dragElement.init = function init(options) {
 
         if(!gd._dragged) {
             var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            e2.initMouseEvent('click', true, true,
```

Comment:
Oh, looks like `initMouseEvent` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent). We should try using `new MouseEvent` first and then fallback to `initMouseEvent` in browsers that don't support it.